### ISSUE TITLE
Hide container color?

### ISSUE THREAD

**Issue by Azkaas** [ranmaru22/firefox-vertical-tabs#104]
Hi!

Is there an option for removing the container color from the tabs?

Thanks

**Comment by hsekitir** [comment]
![2023-06-17-160444_hyprshot](https://github.com/ranmaru22/firefox-vertical-tabs/assets/114614936/399fb4a5-7f5c-466d-8829-d274d26d9f36)

Is there a way to make them thinner?
I cant seem to find anything to adjust the height or width to match the tab properly

**Comment by ranmaru22** [comment]
I assume it's possible with some CSS hack but I haven't looked into it. If you can find a way to do that and make it an optional flag, I would accept a PR for it. 

As for hiding the colors completely: that seems counterintuitive. If you don't use containers, you don't see colors. If you do use them, why would you hide the colors? That way you don't see which tab is in which container, making using them almost pointless.

**Comment by Nocxium** [comment]
Not for my usecase/workflow. I use temporary containers for all sites with no logins, and fixed containers for different categories of websites. So 99% of the time, i know what container a website belongs to since that website is dedicated to that container, and i dont need a color indicator for that.

For people that use multiple colors for the same website, it could be useful though

**Comment by ranmaru22** [comment]
I'm not one to criticize other peoples' workflows. ;)

However, the color indicators are part of the standard tab layout of Firefox, therefore I would assume that most people want them or at least are used to them. I've never tried to hide them away. It could very well be that they're either integrated into the base tab UI (as container tabs used to be a core feature before Mozilla moved it into an extension) or into the core of TCR. 

If there is a way and it can be implemented as an optional flag (CSS var for example) that's unobtrusive, I think we could offer that for the people who want it.

**Comment by Nocxium** [comment]
Yeah, in vanilla firefox it isnt possible without hacks to get it to work afaik. But the previous theme i used, actually had changed the tabs look in a way, that the text got the container color instead of a bar. I still dont know CSS so i cant unfortunatly how they managed to do it, but i was able to find the snippet where they changed the text color, and delete any special color set to it. So the text appeared with the same color as all the other tabs.

Im not gonna ask you to implement this, since it seems like im in the minority and it seems pretty nische :P But if you are interested in finding out how the other plugin achieved what they did, you can find it in rafaelmardojai/firefox-gnome-theme

**Comment by hsekitir** [comment]
May be something like [this](https://www.reddit.com/r/FirefoxCSS/comments/o1ccqe/a_different_way_of_displaying_the_container_color/) would be pretty cool.
